src/common/IconCloseModal.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+
+const IconCloseModal = () => {
+  return (
+    <svg
+      width="24"
```

Comment:
In this way we can use svg images with customizing it, for example icon colors.

Reply:
In my opinion, using` img `is simpler and requires less code compared to` svg`
If you need to use different colors, you can simply use another `img`

Reply:
I agree with you, I have an issue in another place that I was planning to fix the same way (using icon.tsx instead of <img../>), could you give me a tip should we use another method to fix this issue?

Description: when we open wefriiend in the first time (with empty cache) on icon hover we have blinking, because the red icon and the grey icon are separate images:

https://drive.google.com/file/d/1j0AvK7p-r0Kga5CTnkB4GrBPnpz005gN/view?usp=sharing

Reply:
Maybe using `SvgIcon`could help in this case. What do you think?

Reply:
What is the difference between SvgIcon and IconComponents that already used (and this approach existed in the project before)? As I can conclude after research, SvgIcon has more complexity and limits. Anyway I am open to experiments and discussions: we don't have SvgIcon in our project, could you please give an example? As I see it needs changes even in webpack's settings. And as a result converts image files into inline svg. (I use inline svg, in my case there is no need to keep image file(s) additionaly, and only one component can cover all the color/styles requirements.)

Using <img /> doesn't give you "inlining", so svg is not inline and it takes additional html request and makes performance worse. Also if we keep a lot of svg images, that are the same, but differs only with color, we make frontend part heavier (girls just discussed yesterday on meeting why we shouldn't)

Reply:
As far as I know, we haven't used IconComponent in the project yet. If this matters.
In my opinion, there is no need to change the img to IconCloseModal component. It is used only once in the modal window and doesn't change in any way. 
Also, I noticed the iconComponents for chart, location and lightning aren't used in the menu. I think that if you change icons to IconsComponent, it would be better to do it everywhere.

Reply:
Yes, we have used IconComponent before: https://github.com/WeFriiends/WeFriiendsWeb/blob/main/src/common/Lightning.tsx